### ISSUE TITLE
Multiplication following a int8 transposed convolutions isn't constant folded

### ISSUE THREAD

**Issue by lgeiger** [tensorflow/tensorflow#57680]
When converting a model that used int8 quantization aware training, conversion of transposed convolutions followed by a scalar multiplication fails.

The converter isn't able to correctly constant fold the per-tensor fake quantized weights and the scalar multiplication which is a common patter when using transposed convolutions followed by batch normalisation layers. This is a follow-up issue to #53766

### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): macOS / Ubuntu
- TensorFlow installation (pip package or built from source): pip
- TensorFlow library (version, if pip package or github SHA, if built from source): 2.10.0 / 2.11.0-dev20220913

### 2. Code

A minimal reproduction of the issue is available in [this notebook](https://colab.research.google.com/drive/1XM12NAzoiWRJS12uYyzNMsfi571fiauA?usp=sharing). Re-run the notebook to show netron visualisations showing the conversion problem.

**Comment by mohantym** [comment]
Hi @lgeiger !
Thanks for bring it up.
It is replicating in [nightly](https://colab.sandbox.google.com/gist/mohantym/ef92935946f80708c8ce751d45c186bd/quant-transposed-conv-scale.ipynb#scrollTo=hQJCC7lje24_) version as you pointed but not in [2.10 stable release](https://colab.sandbox.google.com/gist/mohantym/2e650b477300cc48b6fef115b826ec10/quant-transposed-conv-scale.ipynb#scrollTo=hQJCC7lje24_). 
Could we stick with 2.10 (stable version) for a while.
Thank you!

**Comment by lgeiger** [comment]
@mohantym Thanks for the fast reply. However, I am able to reproduce this issue in 2.10, in fact this is the version which I am also using locally. Checkout [this notebook](https://colab.research.google.com/drive/1XM12NAzoiWRJS12uYyzNMsfi571fiauA?usp=sharing).

With per-tensor fake quantization I am getting the following TFLite graph:
![Screenshot 2022-09-13 at 16 12 28](https://user-images.githubusercontent.com/13285808/189939377-4ebf3418-f8bf-45e3-8d3b-d8cd559db733.png)
However the multiplication should be fused into the transposed convolution like it is done when not using any weight quantization:
![Screenshot 2022-09-13 at 16 12 35](https://user-images.githubusercontent.com/13285808/189939501-2c5e25ac-f5c6-4644-b0b6-d9ceaccb324b.png)

I get the same result also when re-running your linked notebook.

**Comment by mohantym** [comment]
Hi @lgeiger !
Can you check again with tf-nightly.  I deleted the run time and re-run the code in [nightly](https://colab.sandbox.google.com/gist/mohantym/0f56029ece6175f816c2ffdb8c547f6d/quant-transposed-conv-scale.ipynb#scrollTo=XfH-GKoP1T32)  ( issue seems to be fixed in nightly though , I think it might be specific to mac os )

@sachinprasadhs ! Could you look at this issue. Attached [2.9](https://colab.sandbox.google.com/gist/mohantym/19902aef2b7cd968e3c1f18d85df4652/quant-transposed-conv-scale.ipynb#scrollTo=XfH-GKoP1T32), [2.10](https://colab.sandbox.google.com/gist/mohantym/2e650b477300cc48b6fef115b826ec10/quant-transposed-conv-scale.ipynb#scrollTo=hQJCC7lje24_) and[ nigthly](https://colab.sandbox.google.com/gist/mohantym/0f56029ece6175f816c2ffdb8c547f6d/quant-transposed-conv-scale.ipynb#scrollTo=XfH-GKoP1T32) for reference.

Thank you!

**Comment by lgeiger** [comment]
> Can you check again with tf-nightly. I deleted the run time and re-run the code in [nightly](https://colab.sandbox.google.com/gist/mohantym/0f56029ece6175f816c2ffdb8c547f6d/quant-transposed-conv-scale.ipynb#scrollTo=XfH-GKoP1T32) ( issue seems to be fixed in nightly though , I think it might be specific to mac os )

I double checked by re-running your colab notebook and the issue persists even on nightly `2.11.0-dev20220915`. The scalar multiplication of a transposed convolutions with per-tensor quantized weights is still not fused correctly which results in dequantization and multiplications operation in the graph so that the transposed convolution is executed in float instead of in int8.

**Comment by pjpratik** [comment]
Issue exists in TF 2.11 and TF Nightly 2.12.0-dev20230111 as well. Please find the gist [here](https://colab.research.google.com/gist/pjpratik/e4e0b2879d0d90fb8a95b3ab1df65a97/57680.ipynb). Thank you.

**Comment by pkgoogle** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFLite team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the TFLite space.

**Comment by lgeiger** [comment]
@pkgoogle Thanks for the response. I double checked and the issue still persists in the latest TF nightly. See [this notebook](https://colab.research.google.com/drive/1XM12NAzoiWRJS12uYyzNMsfi571fiauA?usp=sharing).

**Comment by pkgoogle** [comment]
Hi @lgeiger,

Thanks for raising this issue. Are you aware of [AI-Edge-Torch](https://developers.googleblog.com/en/ai-edge-torch-high-performance-inference-of-pytorch-models-on-mobile-devices/)? As we believe this issue is better supported by and more relevant to AI-Edge-Torch we are moving your issue there. Please follow progress [here](https://github.com/google-ai-edge/ai-edge-torch/issues/393).

Let us know if you have any questions. Thanks.
